src/b3d/chisight/gen3d/run_ycbv_evaluation.py

```diff
@@ -49,7 +49,9 @@ def save_hyperparams(folder_name, hyperparams, inference_hyperparams):
         f.write(pprint.pformat(inference_hyperparams))
 
 
-def run_tracking(scene=None, object=None, save_rerun=False, max_n_frames=None):
+def run_tracking(
```

Comment:
I updated this script so we can ask to run it in the mode where it has ground truth pose access.

TODO in the future: when that option is selected, we should probably not do C2F, and just do 1 step of gridding.